docs/features/Docker Support.md

```diff
@@ -1,7 +1,33 @@
 Quartz comes shipped with a Docker image that will allow you to preview your Quartz locally without installing Node.
 
-You can run the below one-liner to run Quartz in Docker.
+You can run Quartz in Docker. A container image is available at:
+
+- `ghcr.io/jackyzha0/quartz`
+
+You can create a local build with:
+
+```sh
+docker build --load -t ghcr.io/jackyzha0/quartz .
+```
+
+This example expects a `content/` directory with the `quartz.*.ts` configuration files and a (empty) `public/` directory to be present in the preesent working directory.
```

Comment:
can you illustrate the working directory structure here?

Reply:
Good suggestion.

My exemplary way to use this container has also changed a bit, as illustrated in my previous comment. I will update the file.

Reply:
I have now added 3b58b2f to address your comment.

From your perspective, does the explanation still lack suitable examples to work with this image?